src/components/RuleList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import CheckIcon from '@mui/icons-material/Check';
+import CloseIcon from '@mui/icons-material/Close';
+
+// Define the type for a single rule
+interface Rule {
+  text: string;
+  met: boolean;
+}
+
+// Define the props for RuleList component
+interface RuleListProps {
+  rules: Rule[];
+}
+
+const RuleList: React.FC<RuleListProps> = ({ rules }) => {
```

Comment:
```suggestion
export const RuleList = (props: RuleListProps)  => {
  const { rules } = props
```
This is easier to read and shorter